corehq/apps/userreports/tasks.py

```diff
@@ -89,9 +89,12 @@ def _build_indicators(config, document_store, relevant_ids):
             adapter.best_effort_save(doc)
 
 
-@task(serializer='pickle', queue=UCR_CELERY_QUEUE, ignore_result=True)
-def rebuild_indicators(indicator_config_id, initiated_by=None, limit=-1, source=None, engine_id=None, diffs=None):
+@serial_task('{indicator_config_id}', default_retry_delay=60 * 10, timeout=3 * 60 * 60, max_retries=5, queue=UCR_CELERY_QUEUE, ignore_result=True)
+def rebuild_indicators(indicator_config_id, initiated_by=None, limit=-1, source=None, engine_id=None, diffs=None, trigger_time=None):
     config = _get_config_by_id(indicator_config_id)
+    if trigger_time is not None and trigger_time > config.last_modified:
```

Comment:
won't `trigger_time` always be greater since it was recorded after the config was saved? What about using the `_rev` and only running the task if the rev passed in is equal to the rev of the config? Alternately pass in `config.last_modified` directly instead of using `utcnow()`.

Final thought, perhaps you meant to say `trigger_time < config.last_modified`